products/compute/api.yaml

```diff
@@ -935,6 +935,154 @@ objects:
         description: |
           How many seconds to wait for the backend before considering it a
           failed request. Default is 30 seconds. Valid range is [1, 86400].
+  - !ruby/object:Api::Resource
+    name: 'RegionBackendService'
```

Comment:
Let's add [`references`](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/products/compute/api.yaml#L57-L61) to preserve docs like https://github.com/terraform-providers/terraform-provider-google-beta/pull/645/files#diff-83aa3ab4e2ca346150e6962897976dbaL12 for readers